test/tools/test_sampcd_processor.py

```diff
@@ -79,11 +83,29 @@ def test_bad_result(self):
             r = _TestResult(name='good', passed=True, bad=True)
 
 
-class Test_get_api_md5(unittest.TestCase):
+class TestSpecFile(unittest.TestCase):
     def setUp(self):
-        self.api_pr_spec_filename = os.path.abspath(
-            os.path.join(os.getcwd(), "..", 'paddle/fluid/API_PR.spec')
+        self.temp_dir = tempfile.TemporaryDirectory()
+        self.api_dev_spec_filename = os.path.join(
+            self.temp_dir.name, 'API_DEV.spec'
+        )
+        self.api_pr_spec_filename = os.path.join(
+            self.temp_dir.name, 'API_PR.spec'
+        )
+        self.api_diff_spec_filename = os.path.join(
+            self.temp_dir.name, 'dev_pr_diff_api.spec'
         )
+        self.init_file()
+
+    def tearDown(self):
+        self.temp_dir.cleanup()
+
+    def init_file(self):
+        raise NotImplementedError
+
+
+class Test_get_api_md5(TestSpecFile):
```

Comment:
```suggestion
class TestGetApiMd5(TestSpecFile):
```

class name 使用 CamelCase